recipe/meta.yaml

```diff
@@ -186,6 +186,7 @@ outputs:
         - kealib
         - arrow-cpp
         - libarchive
+        - libarrow
```

Comment:
You misunderstood (or perhaps I explained badly): you should delete `arrow-cpp`, it has the same content as `libarrow`, but will be removed soon. The issue you're encountering has nothing to do with the renaming of the outputs in conda-forge per se, but with upstream changes around arrow that you need to adapt to.

Reply:
I see; I wasn't sure whether we needed both or just `libarrow`, so I was going to just push one change at a time and see if CI was happy with either 😄